tests/integration/helpers/html-safe-test.js

```diff
@@ -0,0 +1,17 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Helper | html-safe', function(hooks) {
+  setupRenderingTest(hooks);
+
+  // Replace this with your real tests.
```

Comment:
Should we commit tests that are not "real tests"? I don't care about holding up PRs that don't have tests but I don't see why we should commit default tests 🤔